app/src/test/java/io/apicurio/registry/content/ContentCanonicalizerFactoryTest.java

```diff
@@ -171,5 +172,48 @@ void testKafkaConnect() {
         Assertions.assertEquals(expected, actual);
     }
 
+    /**
+    * Test method for {@link io.apicurio.registry.content.ContentCanonicalizerFactory#create(io.apicurio.registry.types.ArtifactType)}.
+    */
+    @Test
+    void testXsd() {
+       ContentCanonicalizerFactory factory = new ContentCanonicalizerFactory();
+       ContentCanonicalizer canonicalizer = factory.create(ArtifactType.XSD);
+
+       ContentHandle content = resourceToContentHandle("xml-schema-before.xsd");
+       String expected = resourceToString("xml-schema-expected.xsd");
+       
+       String actual = canonicalizer.canonicalize(content).content();
+       Assertions.assertEquals(expected, actual);
+    }
+    
+    /**
+     * Test method for {@link io.apicurio.registry.content.ContentCanonicalizerFactory#create(io.apicurio.registry.types.ArtifactType)}.
+     */
+     @Test
+     void testWsdl() {
+        ContentCanonicalizerFactory factory = new ContentCanonicalizerFactory();
+        ContentCanonicalizer canonicalizer = factory.create(ArtifactType.WSDL);
+
+        ContentHandle content = resourceToContentHandle("wsdl-schema-before.wsdl");
+        String expected = resourceToString("wsdl-schema-expected.wsdl");
```

Comment:
Logic is good but please rename the test files to:

* `wsdl-before.wsdl`
* `wsdl-expected.wsdl`

They are not wsdl-schemas, just wsdl. :)